controllers/apiserver.go

```diff
@@ -44,6 +44,9 @@ var samplePipelineTemplates = map[string]string{
 	"sample-config":   "apiserver/sample-pipeline/sample-config.yaml.tmpl",
 }
 
+// If Kubernetes storage is set, DSPO should apply the CRDs
+var kubernetesTemplatesDir = "apiserver/kubernetes"
```

Comment:
How about something like this for clarity?
```suggestion
var kubernetesTemplatesDir = "apiserver/kubernetes-pipeline-store"
```